web/packages/build/package.json

```diff
@@ -10,29 +10,29 @@
     "directory": "packages/build"
   },
   "dependencies": {
-    "@babel/core": "^7.27.1",
+    "@babel/core": "^7.27.7",
     "@babel/preset-env": "^7.27.1",
     "@babel/preset-react": "^7.27.1",
     "@babel/preset-typescript": "^7.27.1",
-    "@eslint/js": "^9.26.0",
-    "@swc/core": "^1.11.24",
+    "@eslint/js": "^9.30.0",
+    "@swc/core": "^1.12.9",
     "@swc/plugin-styled-components": "^7.1.3",
     "@types/jsdom": "^21.1.7",
-    "@vitejs/plugin-react-swc": "^3.9.0",
+    "@vitejs/plugin-react-swc": "^3.10.2",
     "babel-plugin-styled-components": "^2.1.4",
-    "eslint": "^9.26.0",
-    "eslint-plugin-jest": "^28.11.0",
+    "eslint": "^9.30.0",
+    "eslint-plugin-jest": "^28.14.0",
     "eslint-plugin-jest-dom": "^5.5.0",
     "eslint-plugin-react": "^7.37.5",
     "eslint-plugin-react-hooks": "^5.2.0",
-    "eslint-plugin-testing-library": "^7.1.1",
+    "eslint-plugin-testing-library": "7.4.0",
```

Comment:
For some reason dependabot didn't update this lib since ~December. I had to downgrade it from 7.5.3 though because of false positives in one of the rules, see ef572b7db187032a3d.